demos/apps/bookinfo/bookinfo-istio.yaml

```diff
@@ -45,7 +45,7 @@ spec:
             "name": "iptables",
             "image": "istio/init:2017-03-17-22.11.25",
             "imagePullPolicy": "Always",
-            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] } }
+            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] }, "privileged": true }
```

Comment:
We should be careful what permissions we get in the container. Do we really need full privileges? If so, you should remove capabilities.